src/main/python/docs/source/simple_examples.rst

```diff
@@ -0,0 +1,77 @@
+Quickstart
+==========
+
+Let's take a look at some code examples.
+
+Matrix Operations
+-----------------
+
+Making use of SystemDS, let us multiply an Matrix with an scala::
+
+  >>> from systemds.matrix import full  # import full
+  # full generates a matrix completely filled with one number
+  >>> m = full((5, 10), 4.20)  # generate a 5x10 matrix filled with 4.2
+  >>> m_res = m * 3.1  # multiply with scala. Nothing is executed yet!
+  # Do the calculation in SystemDS and print the result (numpy array)
+  >>> print(m_res.compute())
+
+As output we get::
+
+  [[ 13.02  13.02  13.02  13.02  13.02  13.02  13.02  13.02  13.02  13.02]
+   [ 13.02  13.02  13.02  13.02  13.02  13.02  13.02  13.02  13.02  13.02]
+   [ 13.02  13.02  13.02  13.02  13.02  13.02  13.02  13.02  13.02  13.02]
+   [ 13.02  13.02  13.02  13.02  13.02  13.02  13.02  13.02  13.02  13.02]
+   [ 13.02  13.02  13.02  13.02  13.02  13.02  13.02  13.02  13.02  13.02]]
+
+The Python SystemDS package is compatible with numpy arrays.
+Let us do a quick elementwise matrix multiplication of numpy arrays with SystemDS::
+
+  >>> import numpy as np  # import numpy
+  >>> from systemds.matrix import Matrix  # import Matrix class
+
+  # create a random array
+  >>> m1 = np.array(np.random.randint(100, size=5 * 5) + 1.01, dtype=np.double)
+  >>> m1.shape = (5, 5)
+  # create another random array
+  >>> m2 = np.array(np.random.randint(5, size=5 * 5) + 1, dtype=np.double)
+  >>> m2.shape = (5, 5)
+
+  # elementwise matrix multiplication, note that nothing is executed yet!
+  >>> m_res = Matrix(m1) * Matrix(m2)
+  # lets do the actual computation in SystemDS! We get an numpy array as a result
+  >>> m_res_np = m_res.compute()
+  >>> print(m_res_np)
+
+More complex operations
+-----------------------
+
+SystemDS provides high level functions for Data-Scientists, lets take a look at l2svm::
```

Comment:
Could someone improve this sentence? I think @mboehm7 had a good phrase on his slides, but I don't remember it exactly.